### PR TITLE
changed the example of manual propType

### PR DESCRIPTION
**Pull request by decisa**
seems like the example of propTypes should be changed a bit to better match your solution.
for people unfamiliar with propTypes this example will be a better hint to finish the solution by themselves without looking into the final answer.

### CODE REVIEW

**Comment by decisa** [comment]
Well, I just watched the solution video and it describes how you actually got from the example to final solution.
However, if you want students to "come up with a proper solution" on their own, the original example is a bit confusing and hard to follow if you have little programming experience.

Feel free to accept or decline the PR :)

**Review by kentcdodds — APPROVED** [review]
I agree, this is a better "finished" point 👍 

Especially important since that's where I left off with the video. Thank you!

**Comment by kentcdodds** [comment]
@all-contributors please add @decisa for docs
